documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -35,68 +37,68 @@ local template = grafana.template;
         g.row('Discovery')
         .addPanel(
           g.panel('Target Sync') +
-          g.queryPanel('sum(rate(prometheus_target_sync_length_seconds_sum{job=~"$job",instance=~"$instance"}[5m])) by (scrape_job) * 1e3', '{{scrape_job}}') +
+          g.queryPanel('sum(rate(prometheus_target_sync_length_seconds_sum{cluster=~"$cluster",job=~"$job",instance=~"$instance"}[5m])) by (cluster, job, scrape_job, instance) * 1e3', '{{cluster}}:{{job}}:{{scrape_job}}') +
           { yaxes: g.yaxes('ms') }
         )
         .addPanel(
           g.panel('Targets') +
-          g.queryPanel('sum(prometheus_sd_discovered_targets{job=~"$job",instance=~"$instance"})', 'Targets') +
+          g.queryPanel('sum by (cluster, job, instance) (prometheus_sd_discovered_targets{cluster=~"$cluster", job=~"$job",instance=~"$instance"})', '{{cluster}}:{{job}}') +
```

Comment:
This seems to be the line with the biggest change, from a single output to itemized by instance.

Reply:
As the dashboard allows to have multiple clusters, jobs and instances checked, we thought it is just logical to show this panel as well itemized

![Screenshot from 2024-01-09 17-06-06](https://github.com/prometheus/prometheus/assets/2553019/343677a3-a45d-4298-886a-b5e99f6e96c1)

Reply:
Are you fine with this now, @bboreham?

Reply:
Sure. I like for PR descriptions to cover every line changed, and will query any inconsistencies in case the author included that change by mistake.

Reply:
good advice, thanks @bboreham . By doing I was finding, that we didn't included `instance` in the description.

Reply:
changed it accrodingly. @metalmatze can you have another look please?